public/assets/js/collapsible.js

```diff
@@ -0,0 +1,16 @@
+var collapsible = document.getElementsByClassName("collapsible-button");
+var i;
+
+for (i = 0; i < collapsible.length; i++) {
```

Comment:
just to make debugging easier in the future, can we make sure that `i` isn't a global variable that could conflict with for loops in other files?  otherwise looks good!

```suggestion


for (let i = 0; i < collapsible.length; i++) {
```